src/components/OurValues/OurValues.css.js

```diff
@@ -38,14 +41,15 @@ const Value = styled.div`
 	letter-spacing: -0.015em;
 `;
 const Body = styled.div`
-font-family: 'Karla', sans-serif;
-font-size: 16px;
-font-weight: regular;
-line-height: 30px;
-color: #333333
-text-align: center;
-padding: 19px 20px 86px 20px;`;
+  font-family: 'Karla', sans-serif;
+  font-size: 16px;
```

Comment:
In many code bases its considered best practises to express font-size (and sometimes even spacing) in "rem" units where 1 rem = 16px. Obviously not a huge deal here :) But it's good to keep in mind and get used to seeing it :)

Reply:
Thanks Jon for the feedback! I'll update it